winit-core/src/event.rs

```diff
@@ -524,15 +528,8 @@ impl ButtonSource {
     pub fn mouse_button(self) -> MouseButton {
         match self {
             ButtonSource::Mouse(mouse) => mouse,
-            ButtonSource::Touch { .. } => MouseButton::Left,
-            ButtonSource::Unknown(button) => match button {
-                0 => MouseButton::Left,
-                1 => MouseButton::Middle,
-                2 => MouseButton::Right,
-                3 => MouseButton::Back,
-                4 => MouseButton::Forward,
-                _ => MouseButton::Other(button),
-            },
+            ButtonSource::Touch { .. } => MouseButton::LEFT,
+            ButtonSource::Unknown(_) => MouseButton::UNKNOWN,
```

Comment:
The old mapping of `Unknown` codes made no sense since values were entirely unspecified.

However, there were several things mapping to `Unknown(0)`; that *might* need changing.